### PR TITLE
Refactor/additional info

### PR DESCRIPTION
**Pull request by kevinshahfws**

### CODE REVIEW

File: src/openrpc/localization.json [hunk 1]
```diff
@@ -274,7 +274,105 @@
...
       ]   
-    } 
+    },
+    {
+      "name": "addAdditionalInfo",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-manages": [
+            "xrn:firebolt:capability:localization:language"
```
Comment (jlacivita) [inline src/openrpc/localization.json:284]: let's make a new capability for this:
`xrn:firebolt:capability:localization:additional-info`
Also needs to change on the `removeAdditionalInfo` and `additionalInfo` methods.

**Review by jlacivita — CHANGES_REQUESTED** [review]

File: src/openrpc/localization.json [hunk 2]
```diff
@@ -274,7 +274,105 @@
...
+          "name": "value",
+          "summary": "Value to be set for additionalInfo",
+          "schema": {
+            "type": "string"
+          },
+          "required": true
+        }
+      ],
+      "result": {
+        "name": "success",
```
Comment (jlacivita) [inline src/openrpc/localization.json:308]: Probably shouldn't name the parameter 'success'.

I usually name it 'result' in cases like this.

File: src/sdks/manage/sdk.config.json [hunk 3]
```diff
@@ -29,7 +29,8 @@
                 "xrn:firebolt:capability:localization:country-code",
```
Comment (jlacivita) [inline src/sdks/manage/sdk.config.json:29]: We should probably add this to `use` as well, so that Manage Apps can check the value before setting it

**Review by jlacivita — CHANGES_REQUESTED** [review]

File: src/openrpc/localization.json (hunk 2 above)
Comment (kevinshahfws) [reply src/openrpc/localization.json:308]: Makes sense... will change the parameter name for both the methods.

**Comment by kevinshahfws** [comment]
Addressed in PR# 115

**Pull request closed without merging**